db/database.go

```diff
@@ -108,20 +114,24 @@ func (db *Database) GetAllOffers() ([]*types.Offer, error) {
 
 	var offers []*types.Offer
 	for iter.Valid() {
-		key := iter.Key()
+		id := iter.Key()
 
-		// if the key becomes longer than 32, we're not iterating over offers
-		if len(key) > idLength {
+		// if the key/offerID becomes longer than 32, we're not iterating over offers
+		if len(id) > idLength {
 			break
 		}
 
 		encodedOffer := iter.Value()
 		offer, err := types.UnmarshalOffer(encodedOffer)
 		if err != nil {
-			return nil, err
+			log.Warnf("removing invalid offer with ID=0x%X from database: %s", id, err)
+			log.Warnf("invalid offer JSON was: %s", string(encodedOffer))
```

Comment:
I started the log messages with lower case to match you. :)

Reply:
haha nice, thanks!